models/model.js

```diff
@@ -1,4 +1,14 @@
 const db = require("../db/connection");
+const fs = require("fs/promises")
+
+exports.fetchDocumentation = () => {
+    return fs.readFile("./endpoints.json", "utf8", (err, data) => {
```

Comment:
I would include this in the controller and not have a model for this endpoint, since the models are for interacting with the database and this is just accessing a file, it could be done just in the controller